app/_layouts/docs.html

```diff
@@ -109,7 +109,7 @@ <h5>{{nav.title}}</h5>
         <div class="content">
           {% if page.edition == 'community' %}
             <div class="alert alert-ee">
-              Maybe you were looking for the <a href="/enterprise"><strong>Enterprise Edition (EE) Documentation</strong></a> instead?
+              Maybe you were looking for the <a href="/enterprise"><strong>Enterprise Documentation</strong></a> instead?
```

Comment:
"Maybe you were looking for the Kong Enterprise documentation instead?" sounds better